torch/csrc/distributed/c10d/init.cpp

```diff
@@ -13,9 +13,10 @@
 #include <gloo/transport/tcp/device.h>
 #include <pybind11/chrono.h>
 
-#include "torch/csrc/Exceptions.h"
-#include "torch/csrc/utils/object_ptr.h"
-#include "torch/csrc/utils/pybind.h"
+#include <torch/csrc/Exceptions.h>
+#include <torch/csrc/distributed/c10d/ddp.h>
+#include <torch/csrc/utils/object_ptr.h>
+#include <torch/csrc/utils/pybind.h>
```

Comment:
Why did those change to external includes? They are including code internal to this project

Reply:
Do we have a guide for `"header"` being internal and `<header>` being external? It always seemed arbitrary to me :D From a C++ point of view, `"path/to/header"` means relative to the current file and `<path/to/header>` means relative to one of the include directories. Since we use absolute paths for all headers the angle brackets are more correct, but it's more of a stylistic issue than anything else